browser/js/user/user.state.js

```diff
@@ -0,0 +1,51 @@
+app.config(function($stateProvider) {
+
+    $stateProvider.state('user', {
+            url: '/user',
+            templateUrl: 'js/user/templates/user.html',
+            controller: 'UserCtrl',
+            // The following data.authenticate is read by an event listener
+            // that controls access to this state. Refer to app.js.
+            data: {
+                authenticate: true
```

Comment:
good call on using this functionality. we should add this to other state changes that require authentication